rllib/tests/run_regression_tests.py

```diff
@@ -38,7 +38,7 @@
     type=str,
     required=True,
     help="The directory in which to find all yamls to test.")
-parser.add_argument("--num-cpus", type=int, default=5)
+parser.add_argument("--num-cpus", type=int, default=6)
```

Comment:
nooby question,
So I saw in yaml files, num_workers = 0
Here we are setting num-cpus to be 6 - how are 6 of them used?